_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+This strategy’s goal is to define what the target state of the version control systems should be, and provide a roadmap in getting to this target state.
+
+This strategy complements existing IT initiatives, such as [ESDC IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), and supports them with new activities (see [Coherent set of actions](#coherent-set-of-actions)).
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) regarding usage of the Version Control Systems.
+Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see [Coherent set of actions](#coherent-set-of-actions)).
+
+This policy becomes active when IT Products are to be built.
+Once active, all teams involved in the project, and the IT products involved in the IT solution, must comply with this guiding policy.
+
+This Guiding Policy has been prepared by taking into consideration alignment and compliance with existing policy instruments and does not replace them.
+Stakeholders are expected to still comply with existing policy instruments including, but not limited to:
+
+- TBS [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- TBS [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611)
+- ESDC Information Management Policy (being drafted)
+- ESDC Security Policy (being drafted)
+
+### Development and Operations Teams (incl. SSC) <!-- omit in toc -->
+
+1. Use a Git-based Version Control System (VCS) among the following options, in that order of preference:
+   - Public (Unclassified): **[GitHub](https://github.com)** or another [approved public platform (internal)](https://dialogue/grp/TAWS-ATST/Technology_Standards/Brick%20Repository/Software%20Version%20and%20Source%20Code%20Management.mht)
+   - Interdepartmental (up to Protected A): **[GCcode (internal)](https://gccode.ssc-spc.gc.ca)**
+   - Internal (up to Protected B): **[Azure DevOps](https://azure.microsoft.com/en-ca/services/devops/)**
+2. Follow the guidance of [ESDC Open Source Software (OSS) Management Framework (internal)](https://dialogue/grp/TAWS-ATST/Collaboration%20Architecture%20Services/Projects/Artificial%20Intelligence%20and%20Open%20Source/OSS/ESDC%20Open%20Source%20Management%20Framework%20v%201.1.docx).
+3. Leverage the functionalities provided by the selected VCS as much as possible, for instance:
+    - File hosting (source code, scripts, system/database configurations...);
+    - Issues (incl. milestones and Kanban);
+    - Wiki (documentation);
+    - Build and Test Automation (e.g., security-related); and
+    - Deployment to internal or external platforms.
+4. Make their projects in the VCS open by default.
```

Comment:
I'd like to maybe expand on this here so that open by default is properly understood:
- Project space is discoverable
- Teams have established processes to support and encourage contributions from employees outside the team and the organization (public)
- Clear code of conduct is available
- Security Disclosure Process is in place for reporting vulnerabilities and to address them
- Documentation is kept to a minimum but is maintained up to date at all times (anyone should be able to stop by the project and know exactly how to use it.
  - Each release (TBD), documentation is automatically translated (or something like that) 
- Classified information and secrets are kept with secrets management tools/services and never hardcoded in the source code.

Something akin to that.

Reply:
I'd also like to add another rule (or maybe it's outside this strategy...)
- Source code Security and Compliance is automated through the use of key tools and services so that software libraries, dependencies, etc. are assessed every time a new commit to the code base is done.
-  A bill of materials is always automatically generated and made available against the latest commit to the main branch.

Reply:
Awesome points! I added them and also included a link to TBS Guide for Publishing Open Source Code.